docs/setup/Configuration.md

```diff
@@ -41,7 +41,21 @@ The following XML fragment added to your configuration file, will make all your
 ```
 This example is an extract from the demo configuration, for full example configuration file see [config.xml](https://github.com/scenarioo/scenarioo/blob/develop/scenarioo-server/src/main/resources/config-for-demo/config.xml)
 
-**IMPORTANT Remarks**: 
+## Branch Selection List Ordering
+
+The ordering of the Branch Selection is configurable. 
```

Comment:
Please change to: The order of the branch entries in the top level navigation branch selection dropdown is configurable.